src/en/components/card/code.md

```diff
@@ -0,0 +1,31 @@
+---
+title: card - Components
```

Comment:
Can you update the title please to just say `Card`? We are going to change the default generated template and removing `- Components` is one of the things we'll change.